tests/base/polynomials_br.cc

```diff
@@ -0,0 +1,181 @@
+// ---------------------------------------------------------------------
+//
+// HEADER SPACE FOR LATER USE
+//
+//
+//
+//
+//
+//
+//
+//
+//
+//
+// ---------------------------------------------------------------------
```

Comment:
I think you meant to put something here :)

Reply:
Fixed all three test files, thanks!